src/components/Content.js

```diff
@@ -1,37 +1,67 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 
 const Content = ({issues}) => {
 
+    const [isShown, setIsShown] = useState(false);
+    const [author, setAuthor] = useState([])
+
     let count=0,count2 = 0, active, closed;
     active = issues.filter(item => item.state === 'open');
     closed = issues.filter(item => item.state === 'closed');
     count = active.length;
     count2 = closed.length;
+    
+    const loadAuthor = (term) => {
+        console.log(term)
+    }
+
+    useEffect(() => {
+
+    }, [])
+
+    const renderedList = issues.map((issue) => {
+        let labelName = '';
+        
+        issue.labels.map((label) => {
+            labelName = label.name
+         })
```

Comment:
So only 1 label is displayed? What about other labels?

Reply:
eh loop niya sir per title sir

Reply:
@efren02 it should display all tags, not only one